cypress/plugins/index.js

```diff
@@ -0,0 +1,22 @@
+/// <reference types="cypress" />
+// ***********************************************************
+// This example plugins/index.js can be used to load plugins
+//
+// You can change the location of this file or turn off loading
+// the plugins file with the 'pluginsFile' configuration option.
+//
+// You can read more here:
+// https://on.cypress.io/plugins-guide
+// ***********************************************************
+
+// This function is called when a project is opened or re-opened (e.g. due to
+// the project's config changing)
+
+/**
+ * @type {Cypress.PluginConfig}
+ */
+// eslint-disable-next-line no-unused-vars
+module.exports = (on, config) => {
```

Comment:
I think it's ok for the time being to keep this boiler plate - @vidyanambiar we should have a look at any plugins that we use in OCP Console, HAC Core, ConsoleDot etc to see if we could benefit from using them or not.

Reply:
@florkbr I see hac-dev [here](https://github.com/openshift/hac-dev/blob/main/integration-tests/plugins/index.ts) using `plugins/index.js` to set some environment variables in the cypress configuration to be used in test utils. I found some examples of it under 3 plugins in `frontend/packages/*` in the console repo where in addition to setting cypress environment variables, some cypress tasks are set up and there is customization around how the test code is transpiled using cypress/webpack-preprocessor. I don't believe we need these just yet, but we might come across the need for env variables or common tasks as we go through the process of writing specs and then we can update the plugins file. WDYT?

Reply:
> Also, some of these files are empty or do nothing, do we really need them?

@vojtechszocs Yes, as mentioned above they just include some boilerplate code for now. I see plugins used in other projects so I just left these in case we want to build them out more as we start writing tests. I can remove the files if they feel unnecessary at this point. LMK